src/glview/cgal/CGALRenderUtils.cc

```diff
@@ -1,8 +1,37 @@
 #include "CGALRenderUtils.h"
 
+
+int linsystem( Vector3d v1,Vector3d v2,Vector3d v3,Vector3d pt,Vector3d &res,double *detptr)
```

Comment:
Should this rather return a bool
It wouldn't hurt with a comment describing what this function does.